src/app/state/transaction/types.ts

```diff
@@ -32,6 +37,10 @@ export interface Transaction {
   timestamp: number | undefined
   to: string | undefined
   type: TransactionType
+  method: string | undefined
+  runtimeName: string | undefined
+  runtimeId: string | undefined
+  round: number | undefined
```

Comment:
Add a comment that these are undefined on consensus transactions / only appear on paratime transactions

Reply:
Should be resolved.

Reply:
We aren't using `add` and `time`, so those can be ignored.
Make `fee` and `height` undefined, and display them as unknown or display `Round: <round>` as alternative.

How ugly will our codebase get because of this? :/

Reply:
Oh I didn't mean to remove `add` and `time` from API definition. We just shouldn't use some fields when normalizing to `Transaction` type

(I also don't know why TransactionsListRow is needed)